### DISCUSSION TITLE
Welcome to AppStream Discussions!

### DISCUSSION THREAD

**Discussion by ximion** [ximion/appstream#346] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cassidyjames** [comment]
Hey everyone! I'm Cassidy James, co-founder and UX architect at @elementary. I mostly lurk here to keep updated on AppStream happenings as we use it heavily throughout AppCenter (both back- and front-end) and elementary OS itself. :wave:

**Comment by aleixpol** [comment]
Hi! I'm Aleix Pol, Plasma Discover maintainer and @KDE developer as well as currently the KDE e.V. President.
My involvement in AppStream revolves around its integration with Discover and KDE Plasma at large.

I'm also organising the Linux App Summit, in case someone has questions about it. https://linuxappsummit.org
